simtools/layout/layout_array_builder.py

```diff
@@ -0,0 +1,244 @@
+import astropy.units as u
+import matplotlib.patches as mpatches
+import numpy as np
+from astropy.table import QTable
+from matplotlib import pyplot as plt
+from matplotlib.collections import PatchCollection
+
+from simtools import io_handler
+from simtools.util import names
+from simtools.visualization import legend_handlers as leg_h
+
+
+class LayoutArrayBuilder:
+    """
+    Manage the array layout. It provides methods to create and plot a LayoutArray.
+    """
+
+    def __init__(self):
+        """Initialize LayoutArrayBuilder."""
+        self.io_handler = io_handler.IOHandler()
+        self.telescope_types = ["LST", "MST", "SCT", "SST"]
+        telescope_object_list = ["LSTObject", "MSTObject", "SCTObject", "SSTObject"]
```

Comment:
I don't think this and the colours should be defined in the init since it ties the plotting into this class while we actually wanted to keep the plotting as independent as possible so it might move to visualisation.

Reply:
I moved now all things related to plotting to the simtools/visualization/visualize